examples/music-v3-1.py

```diff
@@ -0,0 +1,372 @@
+#!/usr/bin/env python3
+
+"""
+This is an example code that shows how you would setup a simple music bot for Lavalink v3.
+This example is only compatible with the discord.py rewrite branch.
+Because of the F-Strings, you also must have Python 3.6 or higher installed.
+"""
+
+import logging
+import math
+import re
+
+import discord
+import lavalink
+from discord.ext import commands
+
+time_rx = re.compile('[0-9]+')
+
+url_rx = re.compile('https?:\/\/(?:www\.)?.+')  # noqa: W605
+
+
+class Music:
+    def __init__(self, bot):
+        self.bot = bot
+
+        if not hasattr(bot, 'lavalink'):
+            lavalink.Client(bot=bot, loop=bot.loop, log_level=logging.DEBUG)
+
+            bot.lavalink.nodes.add(lavalink.Regions.all(), password='youshallnotpass')
+            self.bot.lavalink.register_hook(self._track_hook)
+
+    def __unload(self):
+        # Clear the players from Lavalink's internal cache
+        self.bot.loop.create_task(self.bot.lavalink.players.safe_clear())

```

Comment:
I think it's better to also call `self.bot.lavalink.destroy()` to safely disconnect websockets.